cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,248 @@
+import { DiscountSchema } from '../../support/utils/discountschema';
+import { ProductCategory } from '../../support/utils/product';
+import { PackingMaterial } from '../../support/utils/packing_material';
+import { PackingInstructions } from '../../support/utils/packing_instructions';
+import { PackingInstructionsVersion } from '../../support/utils/packing_instructions_version';
+import { Builder } from '../../support/utils/builder';
+import { appendHumanReadableNow } from '../../support/utils/utils';
+import { PurchaseOrder, PurchaseOrderLine } from '../../support/utils/purchase_order';
+import {
+  applyFilters,
+  selectNotFrequentFilterWidget,
+  toggleNotFrequentFilters,
+  clearNotFrequentFilters,
+} from '../../support/functions';
+describe('Partial material withdrawal in handling unit editor with Materialentnahmelager', function() {
+  let productForPackingMaterialTU;
+  let productForPackingMaterialLU;
+  let packingMaterialForTU;
+  let packingMaterialForLU;
+  let packingInstructionsNameForTU;
+  let packingInstructionsNameForLU;
+  let productName1;
+  let productCategoryName;
+  let discountSchemaName;
+  let priceSystemName;
+  let priceListName;
+  let priceListVersionName;
+  let productType;
+  let vendorName;
+  let materialentnahme;
+  let warehouseName;
+  let packingInstructionsVersionForTU;
+  let packingInstructionsVersionForLU;
+  let length;
+  let width;
+  let height;
+
+  it('Read the fixture', function() {
+    cy.fixture('logistics/materialentnahme_partial_HU.json').then(f => {
+      productForPackingMaterialTU = appendHumanReadableNow(f['productForPackingMaterialTU']);
+      productForPackingMaterialLU = appendHumanReadableNow(f['productForPackingMaterialLU']);
+      packingMaterialForTU = appendHumanReadableNow(f['packingMaterialForTU']);
+      packingMaterialForLU = appendHumanReadableNow(f['packingMaterialForLU']);
+      packingInstructionsNameForTU = appendHumanReadableNow(f['packingInstructionsNameForTU']);
+      packingInstructionsNameForLU = appendHumanReadableNow(f['packingInstructionsNameForLU']);
+      productName1 = appendHumanReadableNow(f['productName1']);
+      productCategoryName = appendHumanReadableNow(f['productCategoryName']);
+      discountSchemaName = appendHumanReadableNow(f['discountSchemaName']);
+      priceSystemName = appendHumanReadableNow(f['priceSystemName']);
+      priceListName = appendHumanReadableNow(f['priceListName']);
+      priceListVersionName = appendHumanReadableNow(f['priceListVersionName']);
+      productType = f['productType'];
+      vendorName = appendHumanReadableNow(f['vendorName']);
+      materialentnahme = f['materialentnahme'];
+      warehouseName = f['warehouseName'];
+      packingInstructionsVersionForTU = appendHumanReadableNow(f['packingInstructionsVersionForTU']);
+      packingInstructionsVersionForLU = appendHumanReadableNow(f['packingInstructionsVersionForLU']);
+      length = f['length'];
```

Comment:
same remark about length, width, height. if we move into `packing_material.json` we're also getting rid of these 3 variables 🎉